default.py

```diff
@@ -11,6 +11,11 @@
 from operator import itemgetter
 
 import requests
+import requests.packages.urllib3
+
+import cookielib
```

Comment:
Did see that Requests should handle cookies but couldn't get it to work. Using a LWP

Could only get the cookies to work if I imported cookielib and used LWPCookieJAR to persist the cookies into the filesystem as the python script holds no state. 

Will have a look/test again and see if it can be removed/improved.